apps/backend/agents/core/base_agent.py

```
@@ -27,18 +26,18 @@
-    
+
     Features:
     - LangChain 1.0 integration
     - OpenAI GPT-4.1 support
     - Integrated safety guardrails
     - LangSmith tracing
     - COPPA/FERPA compliance
     """
-    
+
     def __init__(
```

Ruff (0.14.4)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)